App.jsx

```diff
@@ -4,25 +4,39 @@ import { createStackNavigator } from '@react-navigation/stack'
 import HomeScreen from './src/components/HomeScreen'
 import { useSelector } from 'react-redux'
 import styles from './src/styles/styles'
+import PostForm from './src/components/PostForm'
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
 
 const Stack = createStackNavigator()
+const Tab = createBottomTabNavigator()
 
-const App = () => {
+const HomeStack = () => {
   const { appTitle } = useSelector(state => state)
   return (
-    <NavigationContainer>
-      <Stack.Navigator>
-        <Stack.Screen
-          name='HomeScreen'
-          component={HomeScreen}
-          options={() => ({
-            title: appTitle,
-            headerStyle: styles.mainHeader,
-            headerTitleStyle: styles.appTitle,
-          })}
-        />
-      </Stack.Navigator>
-    </NavigationContainer>
+    <Stack.Navigator>
+      <Stack.Screen
+        name='HomeScreen'
+        component={HomeScreen}
+        options={() => ({
+          title: appTitle,
+          headerStyle: styles.mainHeader,
+          headerTitleStyle: styles.appTitle,
+        })}
+      />
+    </Stack.Navigator>
+  )
+}
+
+const App = () => {
+  return (
+    <>
+      <NavigationContainer style={{ height: 10 }}>
+        <Tab.Navigator>
+          <Tab.Screen name='Feed' component={HomeStack} />
+          <Tab.Screen name='Post' component={PostForm} />
+        </Tab.Navigator>
+      </NavigationContainer>
+    </>
   )
```

Comment:
Good idea with `HomeStack` component for tab navigation, but we should probably extract it out of the `App` component